pkg/store/storepb/rpc.proto

```diff
@@ -132,6 +138,13 @@ message SeriesResponse {
     /// warning is considered an information piece in place of series for warning purposes.
     /// It is used to warn query customer about suspicious cases or partial response (if enabled).
     string warning = 2;
+
+    /// hints in an opaque data structure that can be used to carry information about how the series
```

Comment:
s/in/is/

Reply:
I would say it simply additional information from store, without specifying what it is. If store wants to send current time in it, it can do that.

Reply:
Thanks @pstibrany for your feedback! Updated 👍